docs/versioned_docs/version-zenith/user-guide/cloud/572923-get-started.mdx

```diff
@@ -0,0 +1,312 @@
+
+1. Update your GitLab CI workflow and add the variable to your CI environment. The environment variable must be named `DAGGER_CLOUD_TOKEN`. Refer to the GitLab documentation on [using CI/CD variables](https://docs.gitlab.com/ee/ci/variables/index.html#use-cicd-variables-in-job-scripts).
+
+1. For **Team** plan subscribers with ephemeral CI runners only: Update your GitLab CI workflow (example below) and adjust the `docker stop` timeout period so that Docker waits longer before killing the Dagger Engine container, to give it more time to push data to the Dagger Cloud cache. Refer to the Docker documentation on the [`docker stop` command](https://docs.docker.com/engine/reference/commandline/stop/).
+
+Here is a sample GitLab CI workflow with the Dagger Cloud integration highlighted:
+
+```yaml title=".gitlab-ci.yml" file=./snippets/get-started/ci/gitlab.yml
+```
+
+</TabItem>
+<TabItem value="CircleCI">
+
+1. Create a new environment variable in your CircleCI project named `DAGGER_CLOUD_TOKEN` and set it to the value of the token obtained in [Step 1](#step-1-sign-up-for-dagger-cloud). Refer to the CircleCI documentation on [creating environment variables for a project](https://circleci.com/docs/set-environment-variable/#set-an-environment-variable-in-a-project).
+
+1. For **Team** plan subscribers with ephemeral CI runners only: Update your CircleCI workflow (example below) and adjust the `docker stop` timeout period so that Docker waits longer before killing the Dagger Engine container, to give it more time to push data to the Dagger Cloud cache. Refer to the Docker documentation on the [`docker stop` command](https://docs.docker.com/engine/reference/commandline/stop/).
+
+1. For GitHub, GitLab or Atlassian Bitbucket source code repositories only: Update your CircleCI workflow and add the following pipeline values to the CI environment. Refer to the CircleCI documentation on [using pipeline values](https://circleci.com/docs/variables/#pipeline-values).
+
+    GitHub:
+
+    ```yaml
+    environment:
+      CIRCLE_PIPELINE_NUMBER: << pipeline.number >>
+    ```
+
+    GitLab:
+
+    ```yaml
+    environment:
+      CIRCLE_PIPELINE_NUMBER: << pipeline.number >>
+      CIRCLE_PIPELINE_TRIGGER_LOGIN: << pipeline.trigger_parameters.gitlab.user_username >>
+      CIRCLE_PIPELINE_REPO_URL: << pipeline.trigger_parameters.gitlab.repo_url >>
+      CIRCLE_PIPELINE_REPO_FULL_NAME: << pipeline.trigger_parameters.gitlab.repo_name >>
+    ```
+
+    Atlassian BitBucket:
+
+    ```yaml
+    environment:
+      CIRCLE_PIPELINE_NUMBER: << pipeline.number >>
+    ```
+
+Here is a sample CircleCI workflow. The `DAGGER_CLOUD_TOKEN` environment variable will be automatically injected.
+
+```yaml title=".circleci/config.yml" file=./snippets/get-started/ci/circle.yml
+```
```

Comment:
just double checking if DAGGER_CLOUD_TOKEN doesn't need to be explicitly set here.

Reply:
@kpenfound can you please confirm?

Reply:
I believe it's correct, circleci just adds in all env variables that are set for the project, for better or worse